scripts/globals/effects/aubade.lua

```diff
@@ -4,12 +4,16 @@
 local effect_object = {}
 
 effect_object.onEffectGain = function(target, effect)
+    target:addMod(xi.mod.SLEEPRES, effect:getPower())
+    target:addMod(xi.mod.LULLABYRES, effect:getPower())
```

Comment:
This reminds me of some similar/related problems that used to have a tracker issue but were closed without resolution several process ago (this is more of a reminder note so I will click resolve after posting)

https://github.com/DarkstarProject/darkstar/issues/2531

tl;dr we're going to need more code dealing with multiple forms of sleep and resistance. Barsleep currently does not check lullaby either, and I do not know if it should. I'd rather not solve all of these by having even more res mods for them and then checking them all.

I think lullaby is legit in that it shoudl have a separate mod, I recall something in game calling it out this way (something that affects lullaby differently form other sleeps) but I can't recall _where_ - for the others it boils down to light or dark tho.

Reply:
yeah sleep is just using one effect regardless of light or dark